db/test/test_data.py

```diff
@@ -72,6 +77,34 @@ def test_submit_low_level_data_missing_keys(self, clean, write, sanity):
             db.data.submit_low_level_data(self.test_mbid, self.test_lowlevel_data, gid_types.GID_TYPE_MBID)
 
 
+    def test_write_low_level_submission_offset(self):
```

Comment:
@alastair This is a test to ensure the offset value is expected. Given that `_get_submission_offset` is a nested function, I was not sure about the best way to use unittestt. Does querying for the offset after writing data each time here seem like an appropriate method?

Reply:
this test is fine, for now let's leave the duplicated code. If we need the same functionality somewhere else we should copy it out in the data module